hpx/runtime/threads/policies/lockfree_queue_backends.hpp

```diff
@@ -85,14 +110,24 @@ struct lockfree_lifo_backend
 
     bool push(const_reference val, bool other_end = false)
     {
+#if defined(HPX_HAVE_CXX11_STD_ATOMIC_128BIT)
         if (other_end)
             return queue_.push_right(val);
         return queue_.push_left(val);
+#else
+        // without 128 atomics this turns into a fifo queue
+        return queue_.push(val);
+#endif
```

Comment:
This is just a reminder that this is subtly broken if items are not pushed to the other end when `other_end = true` (#3166 and #3172). I have nothing against this as long as the flecsi people are aware of this. Otherwise it might be better to disable the lifo queue altogether like the abp queues.

Reply:
Yes, I know. However I think you're right. Disabling might be the best option as leaving it as it is right now does not provide any benefits - it just duplicates functionalities in some obscure way,